tools/clang/tools/dxcompiler/dxcapi.cpp

```diff
@@ -91,11 +93,11 @@ static HRESULT ThreadMallocDxcCreateInstance(
   else if (IsEqualCLSID(rclsid, CLSID_DxcOptimizer)) {
     hr = CreateDxcOptimizer(riid, ppv);
   }
-// Note: The following targets are not yet enabled for non-Windows platforms.
-#ifdef _WIN32
   else if (IsEqualCLSID(rclsid, CLSID_DxcIntelliSense)) {
     hr = CreateDxcIntelliSense(riid, ppv);
   }
+// Note: The following targets are not yet enabled for non-Windows platforms.
+#ifdef _WIN32
```

Comment:
You've fixed the missing definition for `IDxcContainerReflection` and `IDxcLinker`. Maybe it's also okay now to enable them in this `ThreadMallocDxcCreateInstance()` function?